src/main/java/io/cdap/plugin/add/field/MultiFieldAdderConfig.java

```diff
@@ -78,6 +81,13 @@ public void validate(FailureCollector failureCollector) {
     }
   }
 
+  /**
+   * Get fieldValue when either key or value specified as macro.
+   */
+  public String getMacroFieldValue() {
+    return getRawProperties().getProperties().get("fieldValue");
```

Comment:
shouldnt `getRawProperties ` be on config?

Reply:
This is already in config. We wrapped it inside the method because we are using it in multiple places, what is your suggestion?

Reply:
Nit: please make `fieldValue` static constant